Description: Correct pod NAME issue in LedgerSMB/RESTXML/Document/Base.pm
 Is missing a pod NAME entry altogether.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472614&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/RESTXML/Document/Base.pm
+++ b/LedgerSMB/RESTXML/Document/Base.pm
@@ -1,3 +1,11 @@
+=pod
+
+=head1 NAME
+
+LedgerSMB::RESTXML::Document::Base - Base XML:Twig structure function
+
+=cut
+
 package LedgerSMB::RESTXML::Document::Base;
 use strict;
 use warnings;
